index.js

```diff
@@ -6,6 +6,39 @@ import * as DID from '@ipld/dag-ucan/did'
 import { CarWriter } from '@ipld/car'
 import { getClient, checkPathsExist, filesize, readProof, filesFromPaths, uploadListResponseToString } from './lib.js'
 
+export async function accessClaim () {
```

Comment:
Ah probably not - this is the implementation of `w3 can access claim` which, like the whole family of `can` commands, is intended to just execute a single capability. 

I think in the longer run it might make sense to have a couple different modes here:

1) execute this capability in "addProofs" mode (the current default thanks to https://github.com/web3-storage/w3up-client/blob/main/src/capability/access.js#L25) and don't print them to the console - `addProofs` ensures the proofs are saved to the agent's cache so w3cli users can then run `w3 proofs ls` to see all the proofs they've gotten
2) execute this capability in "don't addProofs" mode - in this case it probably makes sense to print the proofs we've received to the console, since they won't go anywhere else

For now I think we should just stick with (1) since that's what's implemented

Reply:
removed the console.log in here